### ISSUE TITLE
Repository Wizard breaks after trying to add a new Repository Manager 

### ISSUE THREAD

**Issue by alainna** [BirkbeckCTP/janeway#3916]
**Describe the bug**
After updating the Repository ``eer-stg`` via the Repository Wizard to add a new Repository Manager, an error was received and the Wizard now inaccessible. The repository can still be updated via django admin panel, and the ``EA-stg`` Repository Wizard hosted on the same Press remains accessible. 

**Janeway version**
1.5.1 RC1

**To Reproduce**

1. Log in with Janeway STG admin account and access repository wizard step 3: ``https://stg.ecoevorxiv.org/repository/wizard/repository/eer-stg/step/3/``

2. Attempt to add a new Repository Manager  

3. Encounter error. (Sentry report: ``https://california-digital-library.sentry.io/issues/4843659763/``, reproduced below)

````
NoReverseMatch: Reverse for 'repository_repository_history' with arguments '('',)' not found. 1 pattern(s) tried: ['admin/repository/repository/(?P<object_id>.+)/history/\\Z']
  File "django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "repository/views.py", line 1570, in repository_wizard
    return render(request, template, context)
  File "django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "django/template/base.py", line 170, in render
    return self._render(context)
  File "django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 195, in render
    return template.render(context)
  File "django/template/base.py", line 172, in render
    return self._render(context)
  File "django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/defaulttags.py", line 519, in render
    return self.nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/defaulttags.py", line 315, in render
    return nodelist.render(context)
  File "django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "django/template/defaulttags.py", line 449, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "django/urls/base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "django/urls/resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
````
**Expected behavior**
Wizard would load sans issue

**Additional context**
Previous updates to Repository Manager lists via the Wizard in 1.5.1. RC1 were without issue. 

@everreau has done some investigating and notes that it seems that the bug has to do with generating the history URLs -- namely, that the the version is not defined in this context. We've commented out what we think is the offending section in the template on our stg server where ``eer-stg`` is, from ``.../src/templates/admin/elements/history_modal.html``

 ````
 {% with rest=history.count|add:"-10" %}
   {% if rest > 0 %}
     <tr>
       <td>
         {% if request.user.is_staff %}
             <a
               href="{% url view_url version.id %}">
              {{ rest }} more version{{ rest|pluralize }}
             </a>
         {% else %}
           {{ rest }} more version{{ rest|pluralize }}
         {% endif %}
       </td>
     </tr>
   {% endif %}
 {% endwith %}
 ````

**Comment by ajrbyers** [comment]
@joemull can you add a patch for this so we can include it in RC-5?

**Comment by joemull** [comment]
Thanks @alainna and @everreau! This was identified in #3863 but we only just today managed to merge the fix #3869 into https://github.com/BirkbeckCTP/janeway/tree/b_1_5_1
